### DISCUSSION TITLE
How can I get the validation loss to understand if my training overfits?

### DISCUSSION THREAD

**Discussion by tmargary** [open-mmlab/mmocr#2008] [Q&A]
In the logs I get the training loss, but how do I make MMOCR also show me the validation loss?
